Sources/NetworkProtection/ExtensionMessage/ExtensionMessage.swift

```diff
@@ -65,7 +65,10 @@ public enum ExtensionMessage: RawRepresentable {
         case .isHavingConnectivityIssues:
             self = .isHavingConnectivityIssues
         case .setSelectedServer:
-            guard data.count > 1 else { return nil }
```

Comment:
This was off, empty data didn't mean to ignore the message... it was a request to reset the selected server to automatic.